template/src/app-header.js

```diff
@@ -0,0 +1,16 @@
+import { registerHtml } from 'tram-one'
+
+/**
+ * Simple component that uses children in the rendered tag.
+ * Read more about it here: https://tram-one.io/#components
+ */
+
+const html = registerHtml()
+
+export default (props, children) => {
+	return html`
+    <h1 class="app-header">
```

Comment:
Is it just the view here, or is the indentation weird here?

Reply:
previously the project inconsistently has tabs and spaces. It should now all be tabs (this will be better enforced with a linting tool).